src/puzzle/Operation.js

```diff
@@ -146,7 +146,6 @@ Operation:{
 		
 		switch(this.property){
 			case 'qcmp': bd.cell.each(function(cell){ if(piece===cell.base){ cell.draw();}}); break;
-			case 'qsub': break;
```

Comment:
This is for the autochecker to fire when we mark a cell unshaded.

Skipping that seems like an unnecessary optimization so I'm not too worried about the change, but it points at semantic confusion, since we're now relying on `qsub` instead of `qans` for the answer.